Servers/UI/OJS.Servers.Ui/ClientApp/src/pages/contest-results/ContestResultsPage.tsx

```diff
@@ -122,6 +138,12 @@ const ContestResultsPage = () => {
         [ contestResults ],
     );
 
+    useEffect(() => {
```

Comment:
Something is probably wrong here. When you enter the web console, we have the following error.
![Screenshot from 2023-09-14 09-50-35](https://github.com/SoftUni-Internal/OpenJudgeSystem/assets/71330136/3ae1d82b-a3a6-45f2-bd44-98303aeec790)

Reply:
Fixed